spec/unleash/toggle_fetcher_spec.rb

```diff
@@ -52,4 +58,31 @@
       end
     end
   end
+
+  describe '.new' do
+    context 'when there are problems fetching toggles' do
+      before do
+        cache_creator = described_class.new
+        cache_creator.toggle_cache = { features: [] }
+        cache_creator.save!
+
+        WebMock.stub_request(:get, "http://toggle-fetcher-test-url/client/features").to_return(status: 500)
+      end
+
+      it 'reads the backup file for values' do
+        expect(toggle_fetcher.toggle_cache).to eq("features" => [])
```

Comment:
Would have been nice to test that `File.read` was called, but I guess that's a bunch of work, when we are pretty sure already that it works here.

Reply:
I would argue that `File.read` or the previous way of reading (File.open+file.read) is maybe more an implementation detail, the test would not care how the data ends up in the correct place.